### PR TITLE
replaced default contact form with formspree.io

### PR DESCRIPTION
**Pull request by maximevaillancourt**
Fixes issue #11.
- replaces the default contact form with formspree.io
- adds a "thank you" page

### CODE REVIEW

File: merci.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <title>Merci! - JDIs</title>
```
Comment (DrPandemic) [inline merci.html]: All the node like `title`, `meta`, `link` and pseudo comments, needs to be in a `head` node and not directly in the `html`.

File: index.html [hunk 2]
```diff
@@ -795,9 +803,13 @@ <h2 class="large section-title">Contactez-nous</h2>
         <script src="js/newsletter.js" type="text/javascript"></script>
         <!--image loads plugin -->
         <script src="js/jquery.imagesloaded.min.js" type="text/javascript"></script>
-        <script src="js/jquery.counterup.min.js" type="text/javascript"></script>
+        mscript src="js/jquery.counterup.min.js" type="text/javascript"></script>
```
Comment (DrPandemic) [inline index.html]: typo, should be `<script>`

File: index.html [hunk 3]
```diff
@@ -704,14 +692,13 @@ <h2 class="large section-title">Contactez-nous</h2>
...
-                                        <textarea rows="5" class="form-control" placeholder="Votre message" id="message" required data-validation-required-message="Please enter a message."></textarea>
+                                        <textarea name="message" rows="5" class="form-control" placeholder="Votre message" id="message" required data-validation-required-message="Veuillez entrer un message."></textarea>
                                         <p class="help-block"></p>
                                     </div>
                                 </div>
-                                <br>
-                                <div id="success"></div>
                                 <div class="row">
                                     <div class="form-group col-xs-12">
+					<input type="hidden" name="_next" value="merci.html" />
```
Comment (DrPandemic) [inline index.html:701]: indentation

File: index.html [hunk 4]
```diff
@@ -678,24 +677,13 @@ <h2 class="large section-title">Contactez-nous</h2>
                     <div class="row">
                         <div class="col-md-8">
                             <h4>Envoyez-nous un message</h4>
-                            <form name="sentMessage" id="contactForm" method="post" novalidate>
+				<form method="POST" action="//formspree.io/info@jdis.ca">
```
Comment (DrPandemic) [inline index.html]: The indentation is wrong here

**Comment by DrPandemic** [comment]
Some minor style issues, but everything works well

**Comment by maximevaillancourt** [comment]
Should be better now.

File: merci.html (hunk 1 above)
Comment (maximevaillancourt) [inline merci.html]: No idea how that happened. I think I copied code from Firefox's inspector, hence the weird html tag classes. Now fixed.

**Comment by DrPandemic** [comment]
LGTM :+1:

**Pull request merged**